zk/model/config.py

```diff
@@ -0,0 +1,37 @@
+""" Configuration key store, elements such as event_city or contact_email """
+import sqlalchemy as sa
+from sqlalchemy import orm
+from sqlalchemy.dialects.postgresql import JSON # Required - unsure why
+from meta import Base, Session, metadata
+
+import logging
+log = logging.getLogger(__name__)
+
+class Config(Base):
+    __tablename__ = 'config'
+    category = sa.Column(sa.types.String, primary_key=True)
+    key      = sa.Column(sa.types.String, primary_key=True)
+    value    = sa.Column(sa.dialects.postgresql.JSON)
+    # NOTE: Non-postgresql requires the use of a decorator
+
+    @classmethod
+    def get(cls, category, key=None):
+        """ Get an entry from the config key store
+
+        Calling form:
+        get(category, key) - Fetches the specified value from the database
+        get(key) - Fetches the value from the database with the general category
+        """
+
+        if key is None:
+            key = category
+            category = 'general'
```

Comment:
Ahh, wasn't aware of *args as an option. Arg swapping is ugly, my goal was to make the call as easy as possible.

Other options:
get(key, category) -- category would be optional, just reads the wrong way to me
get(, key) -- category can be blank but is required, SQLAlchemy uses this method for something
get(category, key) -- require category, so all the general gets will be explicit get('general', key)

My preference would be to use *args though.